pepper/libpepper.py

```diff
@@ -233,7 +239,10 @@ def req(self, path, data=None):
                 f = urlopen(req, context=con)
             else:
                 f = urlopen(req)
-            ret = json.loads(f.read().decode('utf-8'))
+            content = f.read().decode('utf-8')
+            if (self.debug_http):
+                print("\nResponse: {}\n".format(content), file=sys.stderr)
```

Comment:
This should use logger.debug, and not print.  Then it can be printed out when the log level is set to debug.

```
logger.debug('Response: %s', content)
```